components/Contest.tsx

```diff
@@ -11,19 +11,27 @@ interface ContestProps {
   index: number
 }
 
-export default function Contest({ contest }: ContestProps) {
+export default function Contest({ contest, index }: ContestProps) {
   return (
-    <div className='max-w-sm rounded overflow-hidden shadow-lg '>
-      <img className='w-full' src='/images/contest.jpeg' alt='' />
-
-      <div className='px-6 py-4 text-center'>
-        <div className='font-bold text-xl mb-2'>{contest.name}</div>
-        <p className='text-gray-700 text-base'>{contest.description}</p>
-        <p className='text-gray-700 text-base'>
-          <a>End date: </a>
-          {contest.date_end.slice(0, 10)}
-        </p>
-        <p>{contest.status}</p>
+    <div
+      key={index}
+      className='bg-white rounded-lg overflow-hidden shadow-md'
+      style={{
+        gridRowEnd: `span ${Math.ceil(Math.random() * 2)}`,
+      }}
+    >
+      <img
+        src='/images/contest.jpeg'
+        className='h-3/6 w-full h-54 object-cover'
+      />
+      <div className='px-4 py-3'>
+        <h3 className='text-gray-900 font-medium text-lg mb-2'>
```

Comment:
Instead of using mb-2 to create a gap i believe. You can also use in the div gap-2 or space to create the gap you want between elements inside.